### PR TITLE
Admin settings

### PR DESCRIPTION
**Pull request by MBishop17**

### CODE REVIEW

File: content/rancher/v2.x/en/admin-settings/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Users, Global Permissions, and Roles
-weight: 15
+title: Admin Settings
+weight: 1100
```
[the remark this answers is not in the document]
Comment (MBishop17) [reply content/rancher/v2.x/en/admin-settings/_index.md]: I made a mistake. It should have been concept\global-configuration.

File: content/rancher/v2.x/en/admin-settings/rbac/_index.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: RBAC
+weight: 25
+aliases:
+    -/rancher/v2.x/en/concepts/global-configuration/users-permissions-roles/
```
[the remark this answers is not in the document]
Comment (MBishop17) [reply content/rancher/v2.x/en/admin-settings/rbac/_index.md]: addressed with commit 00a816e05ff89e4d12d0f6d6f84cc34d0065e5c3

File: content/rancher/v2.x/en/admin-settings/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MBishop17) [reply content/rancher/v2.x/en/admin-settings/_index.md]: Addressed with these commits: 

- 36d4c27
- ecc2ef0
- de8fe60
- ecc2ef0

File: content/rancher/v2.x/en/admin-settings/authentication/configuring-ad/_index.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 ---
-title: Configuring Active Directory (AD)
-weight: 51
+title: Configuring Active Directory
```
[the remark this answers is not in the document]
Comment (MBishop17) [reply content/rancher/v2.x/en/admin-settings/authentication/configuring-ad/_index.md]: changed with commit 5f659f2

File: content/rancher/v2.x/en/admin-settings/rancher-server-url/_index.md [hunk 4]
```diff
@@ -1,8 +1,16 @@
 ---
 title: Rancher Server URL
 weight: 25
```
[the remark this answers is not in the document]
Comment (MBishop17) [reply content/rancher/v2.x/en/admin-settings/rancher-server-url/_index.md]: fixed with a42b70e

File: content/rancher/v2.x/en/admin-settings/authentication/_index.md [hunk 5]
```diff
@@ -1,7 +1,11 @@
 ---
 title: Authentication
-weight: 10
+weight: 25
+aliases:
+    -/rancher/v2.x/en/concepts/global-configuration/authentication/
+    -/rancher/v2.x/en/task/global-configuration/authentication/
```
[the remark this answers is not in the document]
Comment (MBishop17) [reply content/rancher/v2.x/en/admin-settings/authentication/_index.md]: fixed with cc3a200

**Comment by MBishop17** [comment]
ready for another review.

**Comment by deniseschannon** [comment]
LGTM

**Pull request merged**